webapp/src/utils/translationObjectUtil.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, expect, it } from '@jest/globals';
+
+import { getTranslationsIdText } from './translationObjectUtil';
+
+describe('translationObjectUtil()', () => {
```

Comment:
`translationObjectUtil` is not a function.

```suggestion
describe('translationObjectUtil', () => {
```